src/dvclive/live.py

```diff
@@ -446,6 +446,33 @@ def log_artifact(
                     )
 
     def cache(self, path):
+        if self._inside_dvc_exp:
+            msg = f"Skipping dvc add {path} because `dvc exp run` is running."
+            path_stage = None
+            for stage in self._dvc_repo.index.stages:
```

Comment:
Leaving as a follow-up: We should probably wrap all operations around `dvc_repo` to not fail when a DVC exception is raised.